From: Dinar Valeev <dvaleev@suse.com>
Subject: PPC: Build only vmlinux target
Patch-Mainline: no, tbd (ancient patch)

References: bsc#1070805

The zImage target doesn't build on ppc64le and we only ever
care about the resulting vmlinux file anyway, so drop compilation
of the zImage target for us.

Signed-off-by: Dinar Valeev <dvaleev@suse.com>
Signed-off-by: Alexander Graf <agraf@suse.de>

--- a/arch/powerpc/Makefile
+++ b/arch/powerpc/Makefile
@@ -222,7 +222,7 @@ core-$(CONFIG_PERF_EVENTS)	+= arch/power
 drivers-$(CONFIG_OPROFILE)	+= arch/powerpc/oprofile/
 
 # Default to zImage, override when needed
-all: zImage
+all: vmlinux 
 
 # With make 3.82 we cannot mix normal and wildcard targets
 BOOT_TARGETS1 := zImage zImage.initrd uImage

